tests/test_functional.py

```diff
@@ -22,10 +22,6 @@
 )
 from pylint.utils import HAS_ISORT_5
 
```

Comment:
?

Reply:
I never understood what "exhaustivity tests" means, so the TODO is inapplicable, and I think it might be the ``--enable=one-message --disable=all`` done by Andreas recentely.